package.json

```diff
@@ -79,8 +79,8 @@
     "babel-core": "7.0.0-bridge.0",
     "bundlemon": "^1.0.0",
     "chrome-remote-interface": "0.27.1",
-    "cordova": "7.1.0",
-    "cordova-android": "7.1.4",
+    "cordova": "8.1.2",
+    "cordova-android": "9.1.0",
```

Comment:
We should remove cordova-android from there. It should only be in target mobile no?

Reply:
Oui on fait un `cd src/drive/targets/mobile` avant le `cordova xxx` dans les commandes yarn... je vais tester